conf/configuration.go

```diff
@@ -54,6 +54,7 @@ type JWTConfiguration struct {
 
 // MFAConfiguration holds all the MFA related Configuration
 type MFAConfiguration struct {
+	Enabled                     bool    `default:"false" envconfig:"MFA_ENABLED"`
```

Comment:
Why do we have to specify `envconfig` here? It should just work if you write: `GOTRUE_MFA_ENABLED` instead of `MFA_ENABLED` for this case.

Reply:
Yes, you are right it would also work if we specified `GOTRUE_MFA_ENABLED` I think at point of creating this I wanted a name without the `GOTRUE_`  prefix. Will drop the prefix